samples/config/config.sample

```diff
@@ -71,3 +71,34 @@
 # Keys:
 #   camtopic
 #      Gazebo topic where camera images are published.
+#
+#
+# Section [imgcap]:
+#
+# Keys:
+#   width
+#       Width of the image to be captured in Pixels.
+#       Default: Width of the camera sensor.
```

Comment:
Shouldn't this be a value for the default width, in pixels?

Reply:
Default width is dependent on the camera sensor, default width should be as received from camera. The sensor could be 1080P, 720P , VGA etc

Reply:
Then you should say that, ideally listing the mapping between sensor and width.

Reply:
Fixed.